test/e2e/tests/account/incremental-security.spec.js

```diff
@@ -109,12 +109,10 @@ describe('Incremental Security', function () {
         await driver.switchToWindow(extension);
 
         // should have the correct amount of eth
-        let currencyDisplay = await driver.waitForSelector({
+        await driver.waitForSelector({
```

Comment:
I think we need to come up with a different approach because this does not fail if the text is changed for `waitForSelector`, and it does fail with the previous assertion

Reply:
hey Harika :wave: This will fail if the text changed from '1' to another value. 
I'm not sure I understand your point here :thinking:  Could you clarify what you mean?

Reply:
hey @hjetpoluru :wave: The test does fail if the text is changed.
We are precisely looking that the correct amount of ETH is 1. If this value is changed, the test fails. See below:

https://github.com/user-attachments/assets/dc5a02c9-0f66-4d64-b838-564691f75fa9

Do you mean something else?

Reply:
hey @hjetpoluru :wave: I've seen the test fail if the text is changed from `1`to another value :thinking: Do you see something different?

https://github.com/user-attachments/assets/dc5a02c9-0f66-4d64-b838-564691f75fa9

Reply:
Hey @seaona, thanks for addressing the other issue. I tried changing the value to` 0`, and it did not fail. I will try again later today but if you can check earlier that would be great.

Reply:
thank you @hjetpoluru let me check it and will report back :pray: